### PR TITLE
cpud: add -fstab switch

### PR DESCRIPTION
**Pull request by rminnich**
CPU_FSTAB is a string in fstab(5) format, which allows arbitrary
fstab usage. For multiple mounts, just embed newlines in the string
or see the example below for how to populate it from a file.

If we have a guest VM, we can now use virtio-fs and the performance
improvement is pretty stunning.

You can do this:
CPU_NAMESPACE="" PWD=/ CPU_FSTAB="myfs /mnt virtiofs rw 0 0" cpu v ls /mnt

or
CPU_NAMESPACE="" PWD=/ CPU_FSTAB=${fstab} cpu v ls /mnt

for more complex mounts, if you can figure out how to populate an
environment variable from a file without having newlines converted to
spaces :-)

Current limits: code does not parse the options string, so you can not say
CPU_NAMESPACE="" PWD=/ CPU_FSTAB="myfs /mnt virtiofs defaults 0 0" cpu v ls /mnt

since useful defaults are file-system-type specific.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
don't approve yet, we're going to make this a switch: 
-fstab file
longer FSTAB are hard to get into the command line.
-fstab file will place 
CPU_FSTAB 
in the environment for the cpud to use

**Comment by rminnich** [comment]
ok good to go

File: cmds/cpu/cpu.go [hunk 1]
```diff
@@ -203,16 +204,26 @@ func runClient(host, a, port, key string) error {
...
-func env(s *ossh.Session, envs ...string) {
+func env(s *ossh.Session, envs ...string) error {
 	for _, v := range append(os.Environ(), envs...) {
 		env := strings.SplitN(v, "=", 2)
 		if len(env) == 1 {
 			env = append(env, "")
 		}
 		if err := s.Setenv(env[0], env[1]); err != nil {
-			log.Printf("Warning: s.Setenv(%q, %q): %v", v, os.Getenv(v), err)
+			return fmt.Errorf("Warning: s.Setenv(%q, %q): %v", v, os.Getenv(v), err)
```
Comment (orangecms) [inline cmds/cpu/cpu.go:214]: Should this change be reverted? It looks like a leftover from the env var approach.

Comment (orangecms) [reply cmds/cpu/cpu.go:214]: Huh, now I get it... the env var is abused for marshalling. Was a bit unintuitive first. 😅

**Review by orangecms — APPROVED** [review]

File: cmds/cpu/cpu.go (hunk 1 above)
Comment (rminnich) [reply cmds/cpu/cpu.go:214]: This also passes all environment variables.

Comment (rminnich) [reply cmds/cpu/cpu.go:214]: hence os.Environ()

**Pull request merged**